cmd/namespace-scheduler/README.md

```diff
@@ -0,0 +1,38 @@
+# Namespace Scheulder
+
+The Namespace Scheduler is responsible for watching `kcp` for namespaced resources and scheduling them to a physical cluster location.
+
+The Namespace Scheduler will not split resources like the Deployment Splitter, only schedule all resources in the same namespace to one cluster, and move it to another cluster if the cluster becomes unavailable.
+
+The underlying real clusters will react to the these resources by syncing them, updating status, and syncing that status back to `kcp`.
+
+## Running
```

Comment:
I remember starting out was hard since there were no real common well-worn runners. Can we put this into a script or set of scripts that we all use when we're hacking?

Reply:
(Or a set of scripts that set the env up to allow an end-to-end test)